.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +0,0 @@
-# Pull Request Description
-
-Ensure you have read the contribution guide before filling in a description of the
-pull request, regardless of whether it is complete or a work in progress.
-All Pull Requests should include test case(s) which demonstrates the intended
-behavior of a feature, or a regression test demonstrating that the fix resolves
-the issue.
```

Comment:
Removed this since it only repeats the contribution guidelines, and only partially. Users will be notified of `CONTRIBUTING.md` when first opening a PR: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors

Reply:
Makes sense.